src/spikeinterface/core/waveform_extractor.py

```diff
@@ -399,6 +403,9 @@ def return_scaled(self) -> bool:
     def dtype(self):
         return self._params["dtype"]
 
+    def is_read_only(self) -> bool:
```

Comment:
Another comment.

An advice that I have read is that logical / boolean flags should be designed in such a way that most of the testing you do will test for True. 

Reason being, that putting not in front of flags is a little bit harder to read as you need to keep two things in working memory instead of one (like remembering the sign of an expression that people usually mess up in algebraic high school operations). 

Here most of the tests are for not `is_read_only`. On the other hand, "is_read_only" is the common category  / flag that the OS usually uses so that seem more important here.

Just a comment.

Reply:
We could do `is_not_read_only` instead! That's a good point!